extmod/network_cyw43.c

```diff
@@ -353,6 +353,12 @@ STATIC mp_obj_t network_cyw43_config(size_t n_args, const mp_obj_t *args, mp_map
                 cyw43_ioctl(self->cyw, CYW43_IOCTL_GET_VAR, 13, buf, self->itf);
                 return MP_OBJ_NEW_SMALL_INT(nw_get_le32(buf) / 4);
             }
+            case MP_QSTR_password: {
+                size_t len;
+                const uint8_t *buf;
+                cyw43_wifi_ap_get_password(self->cyw, &len, &buf);
+                return mp_obj_new_str((const char*)buf, len);
```

Comment:
Other ports (eg esp8266, esp32) only allow to set the password, not get it.  I don't think there's a good reason/need to being able to get the password.